### PR TITLE
Handle hero images as post metadata

### LINKED ISSUE #409: Put post "hero image" in metadata in order to remove them from feed
Not sure what is the best way, but some posts have big pictures at the top just to look good.
We should avoid them in the feed.
Setting image url as a metadata might and image should be rendered after rss (probably in react templates).

- http://putaindecode.fr/posts/entreprendre/auto-entrepreneuriat-retour-experiences/
- http://putaindecode.fr/posts/windows/comment-coder-sous-windows/

That would also make upgrade of the design easier (if we want to play with hero images).

### PR DESCRIPTION
**Pull request by madx**
Closes #409

### CODE REVIEW

File: content/posts/entreprendre/auto-entrepreneuriat-retour-experiences/index.md [hunk 1]
```diff
@@ -8,13 +8,12 @@ tags:
 authors:
   - kud
   - MoOx
+hero:
```
Comment (MoOx) [inline content/posts/entreprendre/auto-entrepreneuriat-retour-experiences/index.md]: not sure if "hero" is clear, but it's a start :)

**Comment by MoOx** [comment]
Awesome.
Except for the lint issue https://travis-ci.org/putaindecode/putaindecode.fr/builds/62519020#L479
We should document this somewhere don't you think ? (we can handle that if another issue if you want this merged asap)

**Comment by MoOx** [comment]
Ha nice for the lint :)

**Comment by madx** [comment]
I'll use heroImage and creditUrl instead
Le 14 mai 2015 10:15 AM, "Maxime Thirouin" <notifications@github.com> a
écrit :

**Comment by Nyalab** [comment]
this ~new~ feature is cool, thanks

**Pull request merged**